x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/operator/MultivalueDedupeBytesRef.java

```diff
@@ -146,6 +146,29 @@ public BytesRefBlock dedupeToBlockUsingCopyMissing(BlockFactory blockFactory) {
         }
     }
 
+    /**
+     * Sort values from each position and write the results to a {@link Block}.
+     */
+    public BytesRefBlock sortToBlock(BlockFactory blockFactory, BytesRefBlock orderVal) {
+        try (BytesRefBlock.Builder builder = blockFactory.newBytesRefBlockBuilder(block.getPositionCount())) {
+            for (int p = 0; p < block.getPositionCount(); p++) {
+                int count = block.getValueCount(p);
+                int first = block.getFirstValueIndex(p);
+                switch (count) {
+                    case 0 -> builder.appendNull();
+                    case 1 -> builder.appendBytesRef(block.getBytesRef(first, work[0]));
+                    default -> {
+                        copyAndSort(first, count);
+                        BytesRef orderScratch = new BytesRef();
+                        BytesRef order = orderVal.getBytesRef(orderVal.getFirstValueIndex(p), orderScratch);
+                        writeWork(builder, order.utf8ToString().equalsIgnoreCase("DESC") ? false : true);
```

Comment:
I think you should convert to a boolean earlier - like in the Function.

Reply:
Agreed.